index.html

```diff
@@ -6,176 +6,180 @@
-                        i'd love to help with it.</p>
-                <form action="https://formspree.io/f/{form_id}" methop="post" class="text-me">
-                    <input type="text" class="simple" placeholder="User name" maxlength="30" required>
-                    <input type="email" class="simple" placeholder="Email" maxlength="30" required>
-                    <textarea class="text-area" placeholder="comment" maxlength="500" required></textarea>
-                    <button type="submit" class="project">Get in touch</button>
-                </form>       
-            </section>
+                    <p>
+                        A daily selection of privately personalized reads; no account or sign-ups required.</p>
+                    <ul class="category">
+                        <li class="category-list"><a href="index.html">html</a></li>
+                        <li class="category-list"><a href="index.html">css</a></li>
+                        <li class="category-list"><a href="index.html">javaScript</a></li>
+                    </ul>
+                    <button type="submit" class="project">See Project</button>
+                </section>
+            </div>
         </div>
+        <section class="about">
+            <div class="about-content1">
+                <h1>About Myself</h1>
+                <p>
+                    Hello I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.</p>
+                <p class="connect">LET'S CONNECT</p>
+                <ul class="social-1">
+                    <li><a href="index.html"><img src="image/1.png" alt="social1"/></a></li>
+                    <li><a href="index.html"><img src="image/2.png" alt="social2"/></a></li>
+                    <li><a href="index.html"><img src="image/3.png" alt="social3"/></a></li>
+                    <li><a href="index.html"><img src="image/4.png" alt="social4"/></a></li>
+                    <li><a href="index.html"><img src="image/5.png" alt="social5"/></a></li>
+                </ul>
+                <button type="submit" class="project">Get my resume</button>
+            </div>
+            <div class="about-content2">
+                <ul class="skills">
+                    <li id="list"><a href="index.html">Languages</a></li>
+                    <div class="extra">
+                            <a href="index.html">
+                                <img src="image/Ellipse-1.png" alt="social1" />
+                                <p class="list-4">JavaScript</p>
+                            </a>
+                            <a href="index.html">
+                                <img src="image/Ellipse-3.png" alt="social1" />
+                                <p>HTML</p>
+                            </a>
```

Comment:
- Please work on the text color used for the image below, so it corresponds to that of the Figma design. Make it black `#000`

##### Your design 
![Screenshot from 2022-06-10 08-43-43](https://user-images.githubusercontent.com/58771507/173016945-9fcbed55-025b-4007-ace6-4d89f69eef53.png)

##### Figma design 
![Screenshot from 2022-06-10 08-47-13](https://user-images.githubusercontent.com/58771507/173017046-20355563-18bb-42e4-81e7-c438e4dfe1ab.png)

Reach out to me on slack `@kanu mike chibundu` if you find this feedback confusing